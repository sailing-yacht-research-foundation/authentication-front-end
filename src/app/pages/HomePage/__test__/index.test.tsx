import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { HomePage } from '..';

const shallowRenderer = createRenderer();

describe('<HomePage />', () => {
  it('should render the home page and matchs the snapshot', () => {
    shallowRenderer.render(<HomePage />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});