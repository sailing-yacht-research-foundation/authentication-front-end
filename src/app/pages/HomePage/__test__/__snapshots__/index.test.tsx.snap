// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> should render the home page and matchs the snapshot 1`] = `
<HomePage__Wrapper>
  <VideoSection />
  <BenefitCarousel />
  <PartnerAppCarousel />
  <Footer />
</HomePage__Wrapper>
`;
